frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -41,13 +42,14 @@ export const breadcrumbsForGlobalConfig = (detailsPageKind: string, detailsPageP
   },
 ];
 
-const ItemRow = ({ item }) => {
+const ItemRow = ({ item, isDup }) => {
```

Comment:
Naming nit: I would probably rename `isDup` to `showAPIGroup` since that's really the behavior it controls. I think it makes the meaning more obvious below when we use `ItemRow`.

Reply:
Done.